x-pack/metricbeat/tests/ech/metricbeat_fips_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build ech && requirefips
```

Comment:
[Blocker]
Those are also integration tests, is there any reason to no have them in `integration` and with the `integration` tag as well?

Reply:
Our normal integration tests use docker containers as the environment; these do not.